acados/ocp_nlp/ocp_nlp_dynamics_cont.c

```diff
@@ -499,6 +526,30 @@ void ocp_nlp_dynamics_cont_update_qp_matrices(void *config_, void *dims_, void *
         blasfeo_dveccp(nx1, mem->pi, 0, &mem->adj, nu + nx);
     }
 
+    if (opts->compute_hess)
+    {
+        int hess_index = 0;
+        for (int j = 0; j < nx; j++) {
+            for (int k = j; k < nx; k++) {
+                BLASFEO_DMATEL(&mem->hes, nu+k, nu+j) = work->sim_out->S_hess[hess_index];
```

Comment:
in BLASFEO there are routines to insert/extract/add/whatever a row/column/diagonal/vector to/from a matrix/vector, which are likely to be more efficient (and easier to understand) than accessing one single element at the time with the macro